DTDL/edge/syslog/syslog-source.json

```diff
@@ -0,0 +1,63 @@
+{
+    "@context" : "dtmi:dtdl:context;2",
+    "@id" : "dtmi:io:tributech:source:syslog;1",
+    "@type": "Interface", 
+    "extends": "dtmi:io:tributech:source:edge:base;1",
+    "displayName": "Syslog Source",
+    "contents" : [
+        {
+            "@type": "Property",
```

Comment:
Shouldnt this be one of the listener protocols ? Instead of a additional flag ?

Reply:
agree, I will add TCP_TLS  and DTLS to protocols. DTLS is UDP with TLS